### PR TITLE
#3927 Return process variables values in process-instances endpoint

### PR DESCRIPTION
**Pull request by jsokolowskii**
Saves the variable definition id to query service database and retrieves it in the process-instances endpoint with a query param: variableDefinitions=...
Ref: https://github.com/Activiti/Activiti/issues/3927

### CODE REVIEW

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-liquibase/src/main/resources/config/query/liquibase/changelog/13-alter.oracle.schema.7.4.0.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+alter table process_variable
+  add column definition_id varchar(64);
```
Comment (jesty) [inline activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-liquibase/src/main/resources/config/query/liquibase/changelog/13-alter.oracle.schema.7.4.0.sql]: What do you think if we use a meaningful name like process_variable_definition_id or variable_definition_id, please look at such cases.

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/ProcessVariableCreatedEventHandler.java [hunk 2]
```diff
@@ -72,8 +72,8 @@ private ProcessVariableEntity createProcessVariableEntity(CloudVariableCreatedEv
                                                                          new Date(variableCreatedEvent.getTimestamp()),
                                                                          new Date(variableCreatedEvent.getTimestamp()),
                                                                          null);
-        variableEntity.setValue(variableCreatedEvent.getEntity()
-                                                    .getValue());
+        variableEntity.setValue(variableCreatedEvent.getEntity().getValue());
+        variableEntity.setDefinitionId(variableCreatedEvent.getVariableDefinitionId());
```
Comment (jesty) [inline activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/ProcessVariableCreatedEventHandler.java]: as for the field, can we call it VariableDefinitionId

**Review by jesty — CHANGES_REQUESTED** [review]
just few comments

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-liquibase/src/main/resources/config/query/liquibase/changelog/13-alter.oracle.schema.7.4.0.sql (hunk 1 above)
Comment (jsokolowskii) [reply activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-liquibase/src/main/resources/config/query/liquibase/changelog/13-alter.oracle.schema.7.4.0.sql]: Changed to variable_definition_id

File: activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/ProcessVariableCreatedEventHandler.java (hunk 2 above)
Comment (jsokolowskii) [reply activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/ProcessVariableCreatedEventHandler.java]: Done

**Review by jesty — APPROVED** [review]
LGTM

**Review by miguelruizdev — APPROVED** [review]

**Pull request merged**